packages/react-native-reanimated/android/build.gradle

```diff
@@ -338,14 +269,6 @@ android {
                 "**/libv8executor.so",
         ]
     }
-    tasks.withType(JavaCompile) {
-        compileTask ->
-            compileTask.dependsOn(packageNdkLibs)
-    }
-    configurations {
-        extractHeaders
-        extractSO
-    }
```

Comment:
Leftovers from the times when we used to publish .aar instead of building from source